ulog/README.md

```diff
@@ -53,16 +51,19 @@ For example, the following piece of code:
 ```go
 package main
 
-import "go.uber.org/fx/ulog"
+import (
+  "context"
 
+  "go.uber.org/fx/ulog"
+)
 func main() {
-  log := ulog.Logger()
-  log.Info("My info message")
-  log.Info("Info with context", "customer_id", 1234)
+  log := ulog.Logger(context.Background())
```

Comment:
This shouldn't be a pattern we advertise - it's faster and functionally identical to just use `zap.L()`. We shouldn't encourage allocating a context just to access an already-available global.

Also, `ulog.Logger` returns a `*zap.Logger`. If you want a sugared logger, you have to use `ulog.Sugar`. Why not make these examples using the built-in testing package? Then you'll at least be guaranteed that they compile.